wapi/curves.py

```diff
@@ -757,7 +775,8 @@ def get_tags(self):
         return self._load_data(url, 'Failed to fetch tags')
 
     def search_instances(self, tags=None, issue_date_from=None, issue_date_to=None,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Method "search_instances" has 19 parameters, which is greater than the 15 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.dev.wattsight.com/coding_rules#rule_key=python%3AS107)